src/scripts/build_all_repos.sh

```diff
@@ -1182,10 +1184,17 @@ function test_rpms() {
         done
     done
 
+    rpmlint_cmd="rpmlint"
+    rpmlint_version="$(rpmlint --version)"
```

Comment:
better make it big more future proof using 
```
read maj min subm <<< $(rpmlint --version | tr . ' ')
if [ $maj -gt 2 || ($maj -eq 2 && $min -ge 4 ]] ....
```

but i see that in older version print bunch of text first. anyway up to you

Reply:
It's a reasonable suggestion, done.